grails-app/i18n/messages.properties

```diff
@@ -1425,6 +1425,18 @@ localization.enable.label=Enable localization mode
 localization.disable.label=Disable localization mode
 localization.invalid.custom.message=To enable database storage for custom translations, please add <code>openboxes.locale.custom.enabled = true</code> to your <code>openboxes-config.properties</code> file.
 localizations.label=Localizations
+# Locales
+locale.ach.label=Acoli
```

Comment:
One minor change. I've been misspelling this for months. Could you change it to Acholi.